### PR TITLE
Docs: apply sentence-style capitalization

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

Docs: 
- apply sentence-style capitalization

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/automation/trigger.markdown [hunk 1]
```diff
@@ -971,7 +971,7 @@ automation:
       event: sunset
 ```
 
-## Multiple Entity IDs for the same Trigger
+## Multiple entity IDs for the same Trigger
```
Comment (c0ffeeca7) [inline source/_docs/automation/trigger.markdown]:
```suggestion
## Multiple entity IDs for the same trigger
```

File: source/_docs/authentication/providers.markdown [hunk 2]
```diff
@@ -53,7 +53,7 @@ homeassistant:
 
 If you don't specify any `auth_providers` section in the `configuration.yaml` file then this provider will be set up automatically.
 
-### Trusted Networks
+### Trusted networks
 
 The Trusted Networks auth provider defines a range of IP addresses for which no authentication will be required (also known as "allowlisting"). For example, you can allowlist your local network so you won't be prompted for a password if you access Home Assistant from inside your home.
```
Comment (ludeeus) [inline source/_docs/authentication/providers.markdown]:
```suggestion
The trusted networks auth provider defines a range of IP addresses for which no authentication will be required (also known as "allowlisting"). For example, you can allowlist your local network so you won't be prompted for a password if you access Home Assistant from inside your home.
```

File: source/_docs/authentication/providers.markdown [hunk 3]
```diff
@@ -152,7 +152,7 @@ homeassistant:
 
 Assuming you have only the owner created though onboarding process, no other users ever created. The above example configuration will allow you directly access Home Assistant main page if you access from your internal network (192.168.0.0/24) or from localhost (127.0.0.1). If you get a login abort error, then you can change to use Home Assistant Authentication Provider to login, if you access your Home Assistant instance from outside network.
 
-### Command Line
+### Command line
 
 The Command Line auth provider executes a configurable shell command to perform user authentication. Two environment variables, `username` and `password`, are passed to the command. Access is granted when the command exits successfully (with exit code 0).
```
Comment (ludeeus) [inline source/_docs/authentication/providers.markdown]:
```suggestion
The command line auth provider executes a configurable shell command to perform user authentication. Two environment variables, `username` and `password`, are passed to the command. Access is granted when the command exits successfully (with exit code 0).
```

File: source/_docs/automation/templating.markdown [hunk 4]
```diff
@@ -78,7 +78,7 @@ These are the properties available for a [MQTT trigger](/docs/automation/trigger
 | `trigger.payload_json` | Dictionary of the JSON parsed payload.
 | `trigger.qos` | QOS of payload.
 
-### Numeric State
+### Numeric state
 
 These are the properties available for a [Numeric State trigger](/docs/automation/trigger/#numeric-state-trigger).
```
Comment (ludeeus) [inline source/_docs/automation/templating.markdown]:
```suggestion
These are the properties available for a [numeric state trigger](/docs/automation/trigger/#numeric-state-trigger).
```

File: source/_docs/automation/templating.markdown [hunk 5]
```diff
@@ -146,7 +146,7 @@ These are the properties available for a [Time trigger](/docs/automation/trigger
 | `trigger.platform` | Hardcoded: `time`
 | `trigger.now` | DateTime object that triggered the time trigger.
 
-### Time Pattern
+### Time pattern
 
 These are the properties available for a [Time Pattern trigger](/docs/automation/trigger/#time-pattern-trigger).
```
Comment (ludeeus) [inline source/_docs/automation/templating.markdown]:
```suggestion
These are the properties available for a [time pattern trigger](/docs/automation/trigger/#time-pattern-trigger).
```

File: source/_docs/automation/templating.markdown [hunk 6]
```diff
@@ -155,7 +155,7 @@ These are the properties available for a [Time Pattern trigger](/docs/automation
 | `trigger.platform` | Hardcoded: `time_pattern`
 | `trigger.now` | DateTime object that triggered the time_pattern trigger.
 
-### Persistent Notification
+### Persistent notification
 
 These properties are available for a [Persistent Notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
```
Comment (ludeeus) [inline source/_docs/automation/templating.markdown]:
```suggestion
These properties are available for a [persistent notification trigger](/docs/automation/trigger/#persistent-notification-trigger).
```

File: source/_docs/automation/trigger.markdown [hunk 7]
```diff
@@ -685,7 +685,7 @@ automation:
       at: "15:32:00"
 ```
 
-### Input Datetime
+### Input datetime
 
 The Entity ID of an [Input Datetime](/integrations/input_datetime/).
```
Comment (ludeeus) [inline source/_docs/automation/trigger.markdown]:
```suggestion
The entity ID of an [input datetime](/integrations/input_datetime/).
```

File: source/_docs/automation/yaml.markdown [hunk 8]
```diff
@@ -262,7 +262,7 @@ If you want to migrate your manual automations to use the editor, you'll have to
 
 {% endraw %}
 
-### Deleting Automations
+### Deleting automations
 
 When automations remain visible in the Home Assistant Dashboard, even after having deleted in the YAML file, you have to delete them in the UI.
```
Comment (ludeeus) [inline source/_docs/automation/yaml.markdown]:
```suggestion
When automations remain visible in the Home Assistant dashboard, even after having deleted in the YAML file, you have to delete them in the UI.
```

File: source/_docs/configuration/securing.markdown [hunk 9]
```diff
@@ -13,7 +13,7 @@ Here's the summary of what you *must* do to secure your Home Assistant system:
   - **Note**: Storing secrets in `secrets.yaml` does not encrypt them.
 - Regularly keep the system up to date
 
-## Remote Access
+## Remote access
 
 If you want secure remote access, the easiest option is to use [Home Assistant cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about/) the founders of Home Assistant.
```
Comment (ludeeus) [inline source/_docs/configuration/securing.markdown]:
```suggestion
If you want secure remote access, the easiest option is to use [Home Assistant Cloud](/cloud/) by which you also [support](https://www.nabucasa.com/about/) the founders of Home Assistant.
```
Generally goes against the rest of this PR, but that is the name of it shown on <https://nabucasa.com/>

File: source/_docs/frontend/icons.markdown [hunk 10]
```diff
@@ -9,13 +9,13 @@ description: "Material Design Icons in the Home Assistant frontend"
...
 
 Every entity in Home Assistant has a default icon assigned to it. There are way too many to list out here, but you'll see them in your dashboard. You can [customize any of your entities](/docs/configuration/customizing-devices/#icon) to change the icons displayed to you.
 
-## Finding Icons
+## Finding icons
 
-### Icon Picker
+### Icon picker
 
 The most common way you can find icons is by using the icon picker built right into Home Assistant. Click into the "Icon" field when customizing an entity and start typing. The list will filter to icons that match your search criteria. You can also scroll through all available icons when the field is empty.
```
Comment (ludeeus) [inline source/_docs/frontend/icons.markdown]:
```suggestion
The most common way you can find icons is by using the icon picker built right into Home Assistant. Select the "Icon" field when customizing an entity and start typing. The list will filter to icons that match your search criteria. You can also scroll through all available icons when the field is empty.
```

**Review by ludeeus — APPROVED** [review]

**Pull request merged**